.github/workflows/rpk-build.yml

```diff
@@ -41,11 +41,6 @@ jobs:
       with:
         node-version: '12.16.1'
 
-    - name: Generate WASM deps
```

Comment:
reorder your commits, so this is the first commit.

Reply:
Done :+1: